src/ui/server/routes/insurance/index.ts

```diff
@@ -11,6 +11,7 @@ import { post as typeOfPolicySaveAndBackPost } from '../../controllers/insurance
 import { get as pageNotFoundGet } from '../../controllers/insurance/page-not-found';
 import insuranceEligibilityRoutes from './eligibility';
 import insuranceBusinessRouter from './business';
+import yourBuyerRouter from './your-buyer-details';
```

Comment:
The file should be called the same, `your-buyer`

Buyer details is a page/route within the "your buyer" section